### DISCUSSION TITLE
Change HP Path from Default

### DISCUSSION THREAD

**Discussion by ChaoticRune** [mclemente/healthEstimate#242] [Q&A]
Bit of an odd ask, but is there a way to use the JS Rule to change the HP Path used by Health Estimate? I'm trying to change the path it uses for Pathfinder 1e to "actor.system.attributes.wounds" as I'm testing out PF1's optional Wounds & Vigor rule which changes the way HP works*, and I'd like it to measure a creature's Wounds rather than their normal HP.

*Basically it gives everyone two pools of "HP": **Vigor** (Represents superficial injury, is easily healed, and "protects" your Wounds value) and **Wounds** (Represents serious injury, is hard to heal, and will penalize or kill you when its depleted to certain thresholds).

**Comment by mclemente** [comment]
The module already handles Wounds & Vigor if it is enabled, be it for PCs or NPCs.

**Comment by ChaoticRune** [comment]
Sorry you're right, I had just assumed W&V wasn't implemented since PF1's optional systems tend to be a bit overlooked in Foundry 😅

That said, is there a way for me to alter it so it measures off of Wounds alone? I've been looking into using a homebrewed variant of W&V that I found online that divides Wounds into 25% quarters and applies additional effects when you hit each percentage. I've got the additional effects all automated, but I was hoping I would be able to use Health Estimate to make their Wounds percentages more transparent for my players. Being able to mouse over a token and have the game say if they're currently below 75%/50%/25% Wounds is a lot easier than pulling out a calculator lol

I understand if its probably not possible though, it is kind of a niche thing I'm trying to do
